test/gtest/uct/ib/test_ib.cc

```diff
@@ -392,7 +392,7 @@ class test_uct_ib_gid_idx : public test_uct_ib_with_specific_port {
         }
 
         /* check if the gid is valid to use */
-        if (uct_ib_device_is_gid_raw_empty(gid.raw)) {
+        if (uct_ib_device_is_invalid_gid(gid.raw)) {
```

Comment:
it seems in both places we use this function, we have "union ibv_gid" structure. we can change the function to accept "const union ibv_gid *"

Reply:
done